src/recon.rs

```diff
@@ -913,7 +919,7 @@ unsafe fn decode_coefs<BD: BitDepth>(
                     }
                 }
             }
-            1 => {
+            TxClass::H => {
                 let lo_ctx_offsets = None;
                 let stride: ptrdiff_t = 16 as c_int as ptrdiff_t;
                 let shift: c_uint = ((*t_dim).lh as c_int + 2) as c_uint;
```

Comment:
here the changes start repeating, because the C source uses a large macro for the switch cases here. That macro captures a lot of values. Or, well, it's C, so it does not capture anything, but in rust we'd need to pass all the values in somehow.

That'll be messy, so I think it's best to leave that for the future